docs/citation.rst

```diff
@@ -0,0 +1,6 @@
+.. Citation
+
+Citation
+=====
+
+If you use Quokka or the numerical methods implemented, please cite: Wibking and Krumholz 2023.
```

Comment:
Can you add an inline bibtex citation following this:
https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#role-cite-t